RNWCPP/ReactWin32.nuspec

```diff
@@ -15,17 +15,17 @@
     <file src="target\x64\Ship\React.Windows.Desktop.DLL\react-native-win32.*" target="lib\ship\x64" exclude="**\*.iobj;**\*.ipdb;**\*.exp;**\*.ilk" />
 
     <file src="inc\cxxreact\*" target="inc\cxxreact"/>
+    <file src="inc\jsi\*" target="inc\jsi"/>
+    <file src="inc\jschelpers\*" target="inc\jschelpers"/>
     <file src="inc\Yoga\*.*" target="inc\Yoga"/>
     <file src="inc\folly\**\*.*" target="inc" />
     <file src="inc\stubs\**\*.*" target="inc" />
 
     <file src="inc\ReactWin32\msoFolly.h" target="inc\msoFolly"/>
-    <file src="inc\ReactWindowsCore\AbiSafe.h" target="inc"/>
     <file src="inc\ReactWindowsCore\DevSettings.h" target="inc"/>
     <file src="inc\ReactWindowsCore\INativeUIManager.h" target="inc"/>
     <file src="inc\ReactWindowsCore\InstanceManager.h" target="inc"/>
     <file src="inc\ReactWindowsCore\IReactRootView.h" target="inc"/>
-    <file src="inc\ReactWindowsCore\JSBigAbiString.h" target="inc"/>
```

Comment:
I don't see AbiSafe.h and JSBigAbiString.h being removed elsewhere in your dpk?  these headers are currently used in devmain (reactnativehost) and needed in the nuspec